lib/validator/v3/validator.go

```diff
@@ -427,6 +429,49 @@ func validateCIDR(v *validator.Validate, topStruct reflect.Value, currentStructO
 	return err == nil
 }
 
+// validateHTTPMethods validate if the HTTP Method match clauses are valid.
+func validateHTTPMethods(v *validator.Validate, topStruct reflect.Value, currentStructOrField reflect.Value, field reflect.Value, fieldType reflect.Type, fieldKind reflect.Kind, param string) bool {
+	validMethods := []string{"GET", "PUT", "POST", "HEAD", "DELETE", "OPTIONS", "CONNECT", "*"}
+	methods := field.Interface().([]string)
+	for _, method := range methods {
+		match := false
+		for _, v := range validMethods {
+			if strings.Compare(strings.ToUpper(method), v) == 0 {
```

Comment:
Should we allow lower case here?  I thought we required the correct casing elsewhere, and it means that Felix/Dikastes need to be tolerant to different casing.

Reply:
yes. will change to address this.